.github/workflows/tlaplus.yml

```diff
@@ -12,48 +12,54 @@ on:
 jobs:
   model-checking:
     name: Model Checking
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+    container:
+      image: ccfmsrc.azurecr.io/ccf/ci:05-09-2023-virtual-clang15
 
     steps:
       - uses: actions/checkout@v3
-      - run: python ./tla/install_deps.py
+      - run: |
+          sudo apt update
+          sudo apt install -y default-jre
+          python3 ./tla/install_deps.py
 
       - name: MCccfraft.cfg
         run: |
-          set -exo pipefail
           cd tla/
-          ./tlc.sh -workers auto MCccfraft.tla 2>&1 | tee MCccfraft.out
+          ./tlc.sh -workers auto MCccfraft.tla -dumpTrace tla MCccfraft.trace -dumpTrace json MCccfraft.json
```

Comment:
Can the tla trace file have a .tla filetype? This helps with automatic syntax highlighting etc. The default naming such as `MultiNodeReads_TTrace_1696326252.tla` works nicely for this (also `*TTrace*.tla` is already in the .gitignore)

Reply:
Sure, we just need a pattern that's recognisable for upload, to separate the traces from the specs. Perhaps .trace.tla?